cmd/soroban-cli/Cargo.toml

```diff
@@ -53,6 +53,7 @@ csv = "1.1.6"
 ed25519-dalek = "1.0.1"
 jsonrpsee-http-client = "0.15.1"
 jsonrpsee-core = "0.15.1"
+http = "0.2.9"
```

Comment:
This is not really a new dependency. The `http` dependency is the same dependency that the jsonrpsee package uses for parsing URLs.